core/uemis-downloader.c

```diff
@@ -243,7 +251,8 @@ static bool uemis_init(const char *path)
 {
 	char *ans_path;
 	int i;
-
+	// Wipe mapping of divespot ids and uuids
+	divespot_downloads = NULL;
```

Comment:
`free()` the old list? Either here or at the end of `do_uemis_import()`?

Reply:
Ok. We talk about a tiny amount of memory, but you are right. I will make a change to free the entire list on init.